runtime/syntax/raku.yaml

```
@@ -1,7 +1,7 @@
 filetype: raku
 
 detect:
-    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku$|\\.rakumod$|\\.rakudoc$)"
+    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku(mod|doc|test)?$|\\.nqp$)"
```

The leading `\\.` and trailing `$` can also be factored out to make it bit nicer to read:
```suggestion
    filename: "\\.(p6|pl6|pm6|raku(mod|doc|test)?|nqp)$"
```

---

Yes, good hint and combine `p6|pl6|pm6` to `p(l|m|od)?6`? :wink: